docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+db:
+  image: mongo
+  ports:
+    - "27017:27017"
+
+brigade:
+  build: .
```

Comment:
there's a better way to do this so that we can run incremental builds whenever we make changes to the source code. I'm not that familiar with how to do that, but can revisit that later.